packages/synapse-react-client/src/components/Evaluation/EvaluationRoundEditor.tsx

```diff
@@ -131,10 +133,12 @@ export const EvaluationRoundEditor: React.FunctionComponent<
   }, [error])
 
   const [startDate, setStartDate] = useState<string | Dayjs | null>(
-    dayjs(evaluationRoundInput.roundStart),
+    evaluationRoundInput.roundStart
+      ? dayjs(evaluationRoundInput.roundStart)
+      : null,
```

Comment:
We are now using the built-in MUI validation for the input datetime. Ensure that the starting value is `null` for a new evaluation round, so that the validation error isn't triggered before the user picks a datetime.